src/Microsoft.Health.Dicom.SqlServer/Registration/ServiceCollectionRegistrationExtensions.cs

```diff
@@ -0,0 +1,131 @@
+﻿// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+
+using EnsureThat;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Health.Dicom.Core.Registration;
+using Microsoft.Health.Dicom.SqlServer.Features.Indexing;
+using Microsoft.Health.Dicom.SqlServer.Features.Retrieve;
+using Microsoft.Health.Dicom.SqlServer.Features.ExtendedQueryTag;
+using Microsoft.Health.Dicom.Core.Features.ExtendedQueryTag;
+using Microsoft.Health.SqlServer.Features.Storage;
+using Microsoft.Health.SqlServer.Features.Client;
+using Microsoft.Health.SqlServer;
+using Microsoft.Health.SqlServer.Features.Schema;
+using Microsoft.Health.Dicom.SqlServer.Features.Schema;
+using Microsoft.Health.SqlServer.Features.Schema.Manager;
+using Microsoft.Health.SqlServer.Configs;
+using Microsoft.Health.Dicom.Core.Extensions;
+using System;
+using Microsoft.Health.SqlServer.Registration;
+using Microsoft.Health.SqlServer.Api.Registration;
+using Microsoft.Extensions.DependencyInjection.Extensions;
+using Microsoft.Health.Extensions.DependencyInjection;
+using Microsoft.Health.Dicom.SqlServer.Features.Store;
+using Microsoft.Health.Dicom.Core.Features.Store;
+using Microsoft.Health.Dicom.SqlServer.Features.Query;
+using Microsoft.Health.Dicom.SqlServer.Features.ChangeFeed;
+
+namespace Microsoft.Extensions.DependencyInjection
+{
+    public static class ServiceCollectionRegistrationExtensions
```

Comment:
Has discussed we should reuse the DicomServer one

Reply:
removed